components/QuestionControl.tsx

```diff
@@ -45,9 +54,20 @@ export const QuestionControl = memo<{
 								: "text"
 						}
 						{...(definition.type === "number" ? { min: 0 } : {})}
+						maxLength={
+							Postcode.indexOf(definition.question.toLowerCase()) > -1 ? 5 : 999
+						}
 						required={definition.required}
 						value={value.value ?? ""}
-						onChange={(e) => onChange({ ...value, value: e.target.value })}
+						onChange={(e) =>
+							onChange({
+								...value,
+								value:
+									Postcode.indexOf(definition.question.toLowerCase()) > -1
```

Comment:
nedalo by se misto tohohle proste pouzit `<input type="number">` v pripade ze se ptame na PSC? Cely to je pak jednodussi + na mobilu vyskoci lepsi klavesnice

Reply:
type jde z contemberu